agent/acs/handler/acs_handler.go

```diff
@@ -443,6 +442,7 @@ func (acsSession *session) startACSSession(client wsclient.ClientServer) error {
 			// Once ACS successfully reconnects, set disconnectModeEnabled to FALSE
 			logger.Debug("Turning DisconnectModeEnabled off after successful reconnection.")
 			cfg.SetDisconnectModeEnabled(false)
+			acsSession.taskHandler.ResumeEventsFlow()
```

Comment:
Can you also call acsSession.taskHandler.PauseEventsFlow() when you set disconnectModeEnabled to true? Thanks